doc/api/errors.md

```diff
@@ -644,6 +644,17 @@ according to the encoding provided.
 Used by the `util.TextDecoder()` API when the encoding provided is not one of
 the [WHATWG Supported Encodings][].
 
+<a id="ERR_EVENTS_MAX_LISTENERS_TYPE"></a>
+### ERR_EVENTS_MAX_LISTENERS_TYPE
+
+Used when an attempt is made to set `require('events').defaultMaxListeners` to
+an invalid type.
+
+<a id="ERR_EVENTS_UNHANDLED_ERROR"></a>
+### ERR_EVENTS_UNHANDLED_ERROR
```

Comment:
Do we have to make this specific to the events? I would rather have this in case any unhandled error is thrown / emitted what ever.

Reply:
This is really the only place that we raise such errors that I can find.

Reply:
That is correct for the current situation but for me it is more about trying to be open for future uses.